arch/arm64/core/fpu.c

```diff
@@ -54,9 +63,31 @@ static void DBG(char *msg, struct k_thread *th)
 	k_str_out(buf, p - buf);
 }
 
+static void DBG_PC(char *msg, uintptr_t pc)
+{
+	char buf[80], *p;
+	uintptr_t addr = pc;
+	int i;
+
+	p = dbg_prefix(buf, msg, NULL);
+	strcpy(p, " PC=0x");
+	p += 5;
```

Comment:
" PC=0x" is 6 bytes, so it should be p += 6, right?

Reply:
Obviously 😉